src/Bicep.Core/FileSystem/FileResolver.cs

```diff
@@ -211,10 +211,8 @@ public IEnumerable<Uri> GetFiles(Uri fileUri, string pattern = "")
             return Directory.GetFiles(fileUri.LocalPath, pattern).Select(s => new Uri(s));
         }
 
-        public bool TryDirExists(Uri fileUri)
-        {
-            return fileUri.IsFile && Directory.Exists(fileUri.LocalPath);
-        }
+        public bool DirExists(Uri fileUri) => fileUri.IsFile && Directory.Exists(fileUri.LocalPath);
 
+        public bool Exists(Uri uri) => uri.IsFile && File.Exists(uri.LocalPath);
```

Comment:
[nit] `FileExists` to clarify a bit that this doesn't apply to dirs?

Reply:
Done.